cumulusci/tasks/release_notes/task.py

```diff
@@ -108,96 +109,77 @@ class ParentPullRequestNotes(BaseGithubTask):
-        elif len(pull_requests) > 1:
-            self.logger.info(
-                "More than one pull request returned with base='master' for branch {}".format(
-                    parent_branch_name
-                )
-            )
-            return
-        else:
-            # We can ONLY aggregate child change notes when given the parent_branch option
-            #
-            # We aren't able to append to the 'Unaggregated Pull Requests' header.
-            # We don't know at what time the label was applied to the pull request;
-            # and therefore, we cannot determine which child pull requests are already
-            # aggregated into the parent pull request, and which ones should be
-            # included in the 'Unaggregated Pull Requests' section.
-            generator.aggregate_child_change_notes(pull_requests[0])
-
-    def _get_parent_pull_request(self, branch_name):
+    def _get_parent_pull_request(self):
         """Attempts to retrieve a pull request for the given branch.
         If one is not found, then it is created and the 'Build Change Notes' 
         label is applied to it."""
         requests = get_pull_requests_with_base_branch(
-            self.repo, self.repo.default_branch, branch_name
+            self.repo, self.repo.default_branch, self.branch_name
         )
         if len(requests) == 0:
             self.logger.info(
                 "Pull request not found. Creating pull request for branch: {} with base of 'master'.".format(
-                    branch_name
+                    self.branch_name
                 )
             )
-            parent_pull_request = create_pull_request(self.repo, branch_name)
+            parent_pull_request = create_pull_request(self.repo, self.branch_name)
             add_labels_to_pull_request(
-                self.repo, parent_pull_request, self.build_notes_label
+                self.repo, parent_pull_request, self.options.get("build_notes_label")
             )
         else:
             parent_pull_request = requests[0]
         return parent_pull_request
+
+    def _get_child_branch_name_from_merge_commit(self):
+        pull_requests = get_pull_requests_by_commit(
+            self.github, self.repo, self.project_config.repo_commit
+        )
+        merged_prs = list(filter(is_pull_request_merged, pull_requests))
+
+        child_branch_name = None
+        # TODO is there a case where more than one would be expected?
+        if len(merged_prs) == 1 and merged_prs[0].base.ref == self.branch_name:
+            return merged_prs[0].head.ref
+        return child_branch_name
+
+    def _is_parent_branch_base_branch(self, pull_request):
+        """Returns True if self.branch_name is the base branch of target pull_request"""
+        True if pull_request.base.ref == self.branch_name else False
+
+
+def is_pull_request_merged(pull_request):
+    """Takes a github3.pulls.ShortPullRequest"""
+    return pull_request.merged_at is not None
```

Comment:
Put this in the github utils module to avoid duplicating it between generator.py and task.py